content/documentation/terminology/index.adoc

```diff
@@ -0,0 +1,167 @@
+---
+title: "Terminology"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation"
+---
+
+= Kiali Overview
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Terminology Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/terminology/
+
+== Application
+
+It's a runtime/program that runs in the environment.
```

Comment:
I would refine this to show what is an "application" in Kiali.
Basically it's a logical group of Workloads defined by the label "app".
It can be multiple definitions all right (but I would just focus on what it means for Kiali).